.mailmap

```diff
@@ -1,6 +1,7 @@
 Aron Ahmadia <aron@ahmadia.net>
 Phillip Alderman <phillip.d.alderman@gmail.com>
-James Allen <james@sharelatex.com> <jamesallen0108@gmail.com>
+James Allen <james@sharelatex.com>
```

Comment:
I may have acted too fast in the long process of AUTHORS update in all repos.

The reason for the change is that there were problems in some mailmaps (e.g., with 3 emails following a person name) and these were causing issues when resolving the names.

I'll spent some time thinking about the tradeoffs of different solutions.